eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/email/MailGunEmailService.java

```diff
@@ -0,0 +1,47 @@
+package hu.flowacademy.companycalendar.email;
+
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+@Service
+@RequiredArgsConstructor
+public class MailGunEmailService implements EmailService {
+
+    @NonNull
+    private RESTClient springRestClient;
```

Comment:
Szerintem ehhez nem muszáj új osztályt létrehozni. Nyugodtan húzzuk be ide a `RestTemplate`-et, és használjuk azt.